data/tlg0592/tlg003/tlg0592.tlg003.1st1K-grc1.xml

```diff
@@ -0,0 +1,4377 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+ <title xml:lang="grc">Περὶ εὑρέσεως</title>
+ <author xml:lang="lat">Hermogenes</author>
+<editor>Hugo Rabe</editor>
+ <funder>Harvard Library Arcadia Fund</funder>
+ <principal>Gregory Crane</principal>
+ <respStmt>
+  <persName xml:id="DDD">Digital Divide Data</persName>
+  <resp>Corrected and encoded the text</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Gregory Crane</persName>
+  <resp>Editor-in-Chief, Perseus Digital Library</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Matt Munson</persName>
+  <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Annette Gessner</persName>
+  <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Thibault Clérice</persName>
+  <resp>Lead Developer (University of Leipzig) 2015 - 2017</resp>
+ </respStmt>
+ <respStmt>
+  <persName>Bruce Robertson</persName>
+  <resp>Technical Advisor</resp>
+ </respStmt>
+</titleStmt>
+<publicationStmt>
+<authority>Harvard College Library</authority>
+<idno type="filename">tlg0592.tlg003.1st1K-grc1.xml</idno>
+<availability>
+ <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
+ License</p>
+</availability>
+<date>2017</date>
+<publisher>Harvard College Library</publisher>
+<pubPlace>United States</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="lat">Hermogenis Opera</title>
+  <editor>
+	<persName>
+	 <name>Hugo Rabe</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg0592" xml:lang="lat">Hermogenes</author>
+  <imprint>
+	<publisher>Teubner</publisher>
+	<pubPlace>Leipzig</pubPlace>
```

Comment:
I guess you are right and we don't need a German language xml tag before every German town's name.